extensions/CubesterYT/KeySimulation.js

```diff
@@ -0,0 +1,194 @@
+              },
+            },
+          },
+        ],
+        menus: {
+          KEYS: {
+            acceptReporters: true,
+            items: [
+              { text: "space", value: " " },
+              { text: "up arrow", value: "ArrowUp" },
+              { text: "down arrow", value: "ArrowDown" },
+              { text: "right arrow", value: "ArrowRight" },
+              { text: "left arrow", value: "ArrowLeft" },
+              { text: "enter", value: "Enter" },
+              { text: "backspace", value: "Backspace" },
+              { text: "delete", value: "Delete" },
+              { text: "shift", value: "Shift" },
+              { text: "caps lock", value: "CapsLock" },
+              { text: "scroll lock", value: "ScrollLock" },
+              { text: "control", value: "Control" },
+              { text: "escape", value: "Escape" },
+              { text: "insert", value: "Insert" },
+              { text: "home", value: "Home" },
+              { text: "end", value: "End" },
+              { text: "page up", value: "PageUp" },
+              { text: "page down", value: "PageDown" },
+              "a",
+              "b",
+              "c",
+              "d",
+              "e",
+              "f",
+              "g",
+              "h",
+              "i",
+              "j",
+              "k",
+              "l",
+              "m",
+              "n",
+              "o",
+              "p",
+              "q",
+              "r",
+              "s",
+              "t",
+              "u",
+              "v",
+              "w",
+              "x",
+              "y",
+              "z",
+              "0",
+              "1",
+              "2",
+              "3",
+              "4",
+              "5",
+              "6",
+              "7",
+              "8",
+              "9",
+              "-",
+              ",",
+              ".",
+              "`",
+              "=",
+              "[",
+              "]",
+              "\\",
+              ";",
+              "'",
+              "/",
+            ],
+          },
+        },
+      };
+    }
+
+    async pressKey(args) {
+      const pressKey = new KeyboardEvent("keydown", {
+        key: args.KEY,
+      });
+      document.dispatchEvent(pressKey);
+      await delay(125);
+      const releaseKey = new KeyboardEvent("keyup", {
+        key: args.KEY,
+      });
+      document.dispatchEvent(releaseKey);
+    }
+    async holdKey(args) {
+      const pressKey = new KeyboardEvent("keydown", {
+        key: args.KEY,
+      });
+      document.dispatchEvent(pressKey);
+      await delay(args.SECONDS * 1000);
+      const releaseKey = new KeyboardEvent("keyup", {
+        key: args.KEY,
+      });
+      document.dispatchEvent(releaseKey);
+    }
+    clickMouse(args) {
+      // Placeholder, may remove click blocks due to limitations.
+    }
+    holdMouse(args) {
+      // Placeholder, may remove click blocks due to limitations.
```

Comment:
Why would you add this block if it has no functionality? As Lily said, if your extension isn't finished, draft it.

Reply:
Copy paste comment, refer to the previous one.